config/areas/settings.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+return function ($kirby) {
+    return [
+        'icon'   => 'settings',
+        'label'  => t('view.settings'),
+        'menu'   => true,
+        'routes' => [
+            [
+                'pattern' => 'settings',
+                'action'  => function () use ($kirby) {
+                    return [
+                        'component' => 'k-settings-view',
+                        'props'     => [
+                            'languages' => $kirby->languages()->values(function ($language) {
+                                return [
+                                    'default' => $language->isDefault(),
+                                    'icon' => [
```

Comment:
@bastianallgeier Should we have that `icon` definition actually in the backend? (same with `image`). I guess that should rather be added with the Vue component view.

Reply:
That's true!